google/codelab/provision_spinnaker_codelab_image.sh

```diff
@@ -52,22 +52,27 @@ apt-mark unhold $SPINNAKER_SUBSYSTEMS
 
 # acquire and configure jenkins
 apt-get install -y git
-wget http://pkg.jenkins-ci.org/debian/binary/jenkins_2.1_all.deb
+wget https://pkg.jenkins.io/debian-stable/binary/jenkins_2.89.3_all.deb
```

Comment:
call this version out into a constant at top so it is more easily maintainable

Reply:
Done